app/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/reducer/index.test.ts

```diff
@@ -606,4 +607,14 @@ describe("formBranchesReducer", () => {
       ).toBeTruthy();
     };
   });
+
+  it("setPreventPrefConflicts sets preventPrefConflicts", () => {
+    const oldState = { ...MOCK_STATE };
```

Comment:
Any way to explicitly set the initial state with `preventPrefConflicts` to `false`, just so its explicit here and changes to the default mock later don't break this test?